src/biogeophys/SoilStateType.F90

```diff
@@ -389,11 +389,6 @@ subroutine Restart(this, bounds, ncid, flag)
          interpinic_flag='interp', readvar=readvar, data=this%hk_l_col)
 
      if( use_dynroot ) then
-         call restartvar(ncid=ncid, flag=flag, varname='root_depth', xtype=ncd_double,  &
```

Comment:
Are you pretty confident about this one? If your main confidence comes from the test suite passing (as opposed to tracing the variable through the code), then I'd feel more comfortable if you ran a long restart test with dyn roots (currently the only one in the test suite is a 3-day test). e.g. `ERS_Lm12.f10_f10_musgs.I1850Clm50BgcCrop.cheyenne_intel.clm-clm50dynroots` (that will run a 12-month test and compare it against a 7-month-plus-5-month run; that way we'll be testing restart in the middle of the northern hemisphere growing season; that test _should_ use spun-up initial conditions, but you should confirm that: if it doesn't, we should use a test that does use spun-up initial conditions).

Reply:
Thanks!